frontend/src/queries/schema.ts

```diff
@@ -494,6 +494,9 @@ export interface EventsQuery extends DataNode<EventsQueryResponse> {
     orderBy?: string[]
 }
 
+/**
```

Comment:
there's not a lot of usages of this, so we can remove it completely from the UI right now, though will leave it upto you, might be easier to do that after we've done the initial migration :)

Reply:
I think we still have to keep it in case there are some other places where users can create PersonsNode queries. I'll create a Metabase alert just in case.